docs/getting-started/installation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Installation
+description: Installing Powertools for AWS Lambda
+---
+
+<!-- markdownlint-disable MD043 -->
+
+You can use Powertools for AWS Lambda (TypeScript) by installing it with your favorite dependency management, or via [Lambda Layers](./lambda-layers.md).
+
+The toolkit is compatible with both TypeScript and JavaScript code bases, and supports both CommonJS and ES modules.
+
+All features are available as individual packages, so you can install only the ones you need, for example:
+
+* **Logger**: `npm i @aws-lambda-powertools/logger`{.copyMe}:clipboard:
+* **Metrics**: `npm i @aws-lambda-powertools/metrics`{.copyMe}:clipboard:
+* **Tracer**: `npm i @aws-lambda-powertools/tracer`{.copyMe}:clipboard:
+
+See the [Features](../features/index.md) page for a complete list of available utilities.
+
+### Extra dependencies
+
+Some features use additional dependencies like the AWS SDK for JavaScript v3, which might you need to install separately. Below is a list of utilities that use external dependencies, and the packages you need to install to use them.
```

Comment:
Not related to this PR: Are transitive dependencies not automatically installed / declared as transitive dependencies. What is the reason behind this?

Reply:
For the Parameters utility, we don't know which backend the customer will choose (i.e. SSM, Secrets Manager, AppConfig, etc.) - so we let them install the dependencies they need based on their choice.

For Idempotency, it's mainly future proofing, but once we add different persistence layers like in #3183, the same argument applies.

For Parser, similar to Idempotency, for now we support only Zod as parsing engine - but we have plans on expanding #2431 - so having zod separately follows the pattern above.

Conversely, for other utilities like Tracer or Validation, which have a single strong dependency on X-Ray SDK and Ajv respectively - we install these for you. This is because there's no alternative way of using them, so it makes sense to bundle.

Reply:
Thanks for the explanation 👍